### DISCUSSION TITLE
After Solus update, script not working as before

### DISCUSSION THREAD

**Discussion by lf-araujo** [davatorium/rofi#1553] [Theming]
Hi everyone,

After a system update a script with my rofi configuration stopped working. This is a very old one?

```bash
rofi -show run -combi-modi window,drun,run -location 1 -width 100 \
		 -lines 1 -line-margin 0 -line-padding 1 \
		 -separator-style none -font "mono 10" -columns 9 -bw 0 \
		 -hide-scrollbar \
		 -color-window "#222222, #222222, #b1b4b3" \
		 -color-normal "#222222, #b1b4b3, #222222, #005577, #b1b4b3" \
		 -color-active "#222222, #b1b4b3, #222222, #007763, #b1b4b3" \
		 -color-urgent "#222222, #b1b4b3, #222222, #77003d, #b1b4b3" \
		 -kb-row-select "Tab" -kb-row-tab ""
```

- This was set to show rofi in a line at the top of my screen, but it doesn't anymore. How can I get it to the previous behaviour?

Also, would it be possible for rofi to use Zeitgeist's commonly used files and apps? I miss the behaviour from synapse way back.

**Comment by DaveDavenport** [comment]
What is solus? What is zeitgeist? 
The command you showed uses a very very outdated theme format that has been deprecated a long time (years) ago and been removed recently.

**Reply by DaveDavenport** [reply]
There are many topics in the discussion forum about this, might be a good starting point.

**Reply by lf-araujo** [reply]
Thanks, will look into them. Solus is a linux distro, Zeitgeist is a backend that gnome uses to track recent use of files and apps.

**Reply by DaveDavenport** [reply]
Somebody can make a zeitgeist plugin.
